azure-pipelines-autogen.yml

```diff
@@ -62,7 +62,7 @@ jobs:
   - script: |
       set -Eeuxo pipefail
       cd generator
-      npm run generate-all $(generateParams)
+      npm run generate-all -- --batch-count $(batchCount) --batch-index $(batchIndex)
```

Comment:
>-- [](http://example.com/codeflow?start=27&length=2)

is the additional `--` intentional?

Reply:
Yup, take a look at https://stackoverflow.com/questions/43046885/what-does-do-when-running-an-npm-command. Without it, the `--<param>` args are passed to `npm` rather than to the script that `npm` is executing.

Reply:
Got it, thanks for sharing.